check_process

```diff
@@ -1,40 +1,40 @@
 ;; Test complet
-	; pre-install
-		echo -n "Si j'avais des commandes à exécuter ce serait ici "
-	; Manifest
-		domain="domain.tld"	(DOMAIN)
-		path="/vpnconfig"	(PATH)
-	; Checks
-		pkg_linter=1
-		setup_sub_dir=1
-		setup_root=0
-		setup_nourl=0
-		setup_private=1
-		setup_public=0
-		upgrade=1
-		upgrade=1	from_commit=355b24ea0cd3467d7ba1390ab7d34dd4b2500229
-		upgrade=1   from_commit=1fc458110660ce775f7613091cde3c5fdcfbe4e6
-		backup_restore=1
-		multi_instance=0
-		incorrect_path=1
-		port_already_use=0
-		change_url=0
+    ; pre-install
+        echo -n "Si j'avais des commandes à exécuter ce serait ici "
+    ; Manifest
+        domain="domain.tld" (DOMAIN)
+        path="/vpnconfig"   (PATH)
+    ; Checks
+        pkg_linter=1
+        setup_sub_dir=1
+        setup_root=1
+        setup_nourl=0
+        setup_private=1
+        setup_public=0
+        upgrade=1
+        upgrade=1   from_commit=355b24ea0cd3467d7ba1390ab7d34dd4b2500229
+        upgrade=1   from_commit=1fc458110660ce775f7613091cde3c5fdcfbe4e6
+        backup_restore=1
+        multi_instance=0
+        incorrect_path=1
+        port_already_use=0
+        change_url=0
 ;;; Levels
-	Level 1=auto
-	Level 2=auto
-	Level 3=auto
-	Level 4=na
-	Level 5=auto
-	Level 6=auto
-	Level 7=auto
-	Level 8=0
-	Level 9=0
-	Level 10=0
+    Level 1=auto
+    Level 2=auto
+    Level 3=auto
+    Level 4=na
+    Level 5=auto
+    Level 6=auto
+    Level 7=auto
+    Level 8=0
+    Level 9=0
+    Level 10=0
 ;;; Options
 Email=pitchum@gramaton.org
 Notification=down
 #;;; Upgrade options
-#	; commit=65c382d138596fcb32b4c97c39398815a1dcd4e8
-#		name=Name of this previous version
-#		manifest_arg=domain=DOMAIN&path=PATH&admin=USER&password=pass&is_public=1&
+#   ; commit=65c382d138596fcb32b4c97c39398815a1dcd4e8
+#       name=Name of this previous version
```

Comment:
Maybe we should change the name here ?

Reply:
This is all comments anyway right ?

Reply:
I remove this part, this commit is not tested